InterfaceMATLAB/tsgEvaluateHierarchy.m

```diff
@@ -57,6 +57,9 @@
     end
     if (strcmp(lGrid.sType, 'global') || strcmp(lGrid.sType, 'sequence'))
         [vals] = tsgReadMatrix(sFileO);
+    elseif (strcmp(lGrid.sType, 'fourier'))
+        [tmp] = tsgReadMatrix(sFileO);
+        vals = tmp(:,1:2:end) + sqrt(-1)*tmp(:,2:2:end);
```

Comment:
* why not use MATLAB's own `i`, technically `sqrt(-1)` could be `+i` or `-i`

Reply:
@zbmorrow: let's be consistent with notation, we are using `i` everywhere to write to `cout` and in other functions. Change the loop counter from `i` to `r`.